src/data/profiles.js

```diff
@@ -1,8 +1,24 @@
 const raw = require('./raw');
+const rawNew = require('./raw-new');
 const short = require('short-uuid');
 
+function pickRandom(arr, n) {
+    let result = new Array(n),
+        len = arr.length,
+        taken = new Array(len);
+    if (n > len)
+        throw new RangeError("getRandom: more elements taken than available");
+    while (n--) {
+        const x = Math.floor(Math.random() * len);
```

Comment:
`Math.random()` ini exclusive of 1, jadi perlu dikali (len+1) biar termasuk elemen terakhir.

Reply:
hmm, gitu ya. Bener juga sih